webidl-check

```diff
@@ -127,7 +127,8 @@ def serveRequest():
             except WrongConfigurationException,e:
                 printError(output,doc, textoutputselected, "Could not run widlproc", str(e))
             for header_name in ('Last-Modified', 'Expires'):
-                if copyHeader(sys.stdout.write, srcheaders, header_name):
+                f = lambda header, value: sys.stdout.write("%s: %s" %(header, value))
```

Comment:
sorry to be picky, but if you're going to create a variable for this, you might as well create it outside of the `for` loop :)